terraform/services/api-rds/variables.tf

```diff
@@ -1,9 +1,15 @@
+# Restricting valid "app" values until service has been extended or BCDA and DPC
+
 variable "app" {
   description = "The application name (ab2d, bcda, dpc)"
   type        = string
   validation {
+    condition     = contains(["ab2d"], var.app)
+    error_message = "Valid value for app is ab2d."
+    /*
     condition     = contains(["ab2d", "bcda", "dpc"], var.app)
     error_message = "Valid value for app is ab2d, bcda, or dpc."
+*/
```

Comment:
Let's leave bcda and dpc in the validation. They can still be in the validation but not included in the apply workflow, and this will be one less thing to change once we're applying in those environments.

Reply:
Readded to the validation in latest push